vignettes/release_strategy.Rmd

```diff
@@ -39,34 +39,58 @@ updates via the community meetings, especially any anticipated breaking changes.
 
 # Package Release Process
 
-## Quarterly Release
-
-A package release is done in four parts:
-
-1) Verify that all CI/CD checks and then bundle up and send off to CRAN. See the [chapter](https://r-pkgs.org/release.html#decide-the-release-type) in R Packages for more details.
+## Planned Release
+
+A package release is done in the following steps:
+
+1) Create the release version:
+    a) Create a new feature branch from `main`.
+    a) Update the `Versions:` field in the `DESCRIPTION` file.
+    a) Update `NEWS.md` (replace "(development version)" with the version to be
```

Comment:
These two steps can be done with a call to `usethis::use_version()`. You'll be prompted whether you're making a major, minor, or patch version bump.

Reply:
Thanks, I have combined the two steps into one.